contracts/src/errors/IPCErrors.sol

```diff
@@ -26,11 +26,7 @@ error InvalidBatchSource();
 error InvalidSubnetActor();
 error InvalidCollateral();
 error InvalidConfigurationNumber();
-error InvalidCrossMsgSender();
-error InvalidCrossMsgDstSubnet();
-error InvalidCrossMsgNonce();
-error InvalidCrossMsgValue();
-error InvalidCrossMsgKind();
+error InvalidXnetMessage(string reason);
```

Comment:
Should not we store all the reason strings here and then use them in tests and in the code?